x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/alerts_by_status.tsx

```diff
@@ -230,17 +287,25 @@ export const AlertsByStatus = ({
                           isDonut={true}
                           label={STATUS_OPEN}
                           scopeId={SourcererScopeName.detections}
-                          stackByField="kibana.alert.workflow_status"
+                          stackByField={KIBANA_WORKFLOW_STATUS}
                           timerange={timerange}
                           width={ChartSize}
                         />
                       ) : (
                         <DonutChart
+                          onDonutPartitionClicked={(level) =>
```

Comment:
I might consider wrapping these functions in `useCallback()`. The `DonutChart` and `ChartLabel` will re-render whenever this `AlertsByStatus` component re-renders because of the new function reference. Might not be much of an issue, since these charts _probably should_ update if any of the current props to this component change, but future-proofing _could_ be a good idea here.

Reply:
done!